styles.css

```diff
@@ -1001,3 +987,444 @@ h1 {
+    background-image: url(Images/img3.png);
+  }
+
+  .sup-2 {
+    visibility: visible;
+  }
+
+  .right-block-2 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    position: absolute;
+    width: 369px;
+    height: 257px;
+    left: calc(50% - 369px / 2);
+    bottom: 80px;
+  }
+
+  .action7 {
+    visibility: hidden;
+  }
+
+  .sup12 {
+    position: absolute;
+    top: 0;
+    bottom: 40px;
+  }
+
+  .sup0 {
+    position: absolute;
+    top: 40px;
+  }
+
+  .sup13 {
+    position: absolute;
+    top: 370px;
+  }
+
+  .card-works-3 {
+    position: absolute;
+    left: 900px;
+    right: 0;
+    top: 659px;
+    bottom: 0;
+    border: 0 solid;
+    background-image: url(Images/img4.png);
+  }
+
+  .sup14 {
+    position: absolute;
+    top: 170px;
+  }
+
+  .card-works-4 {
+    position: absolute;
+    width: 369px;
+    height: 435px;
+    left: 476px;
+    top: 1200px;
+    border: 0 solid #091e42;
+    background-image: url(Images/img3.png);
+  }
+
+  .card-works-5 {
+    position: absolute;
+    width: 369px;
+    height: 435px;
+    left: 900px;
+    top: 1200px;
+    border: 0 solid;
+    background-image: url(Images/img4.png);
+  }
+
+  .card-works-6 {
+    position: absolute;
+    width: 369px;
+    height: 435px;
+    left: 55px;
+    top: 1200px;
+    background-image: url(/Images/Mask\ Group.png);
+    background-repeat: no-repeat;
+    background-repeat: 369px 435px;
+    border: 0 solid;
+  }
+
+  .sup23 {
+    position: absolute;
+    top: 370px;
+  }
+
+  .about {
+    position: absolute;
+    width: 100%;
+    height: 1195px;
+    left: 0;
+    top: 2680px;
+    background: #262626;
+    background-image: url(Images/bg7.png);
+    background-repeat: no-repeat;
+    background-position: 100% 20%;
+  }
+
+  .primery {
+    width: 566px;
+    height: 78px;
+    margin-left: 142px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .desc {
+    width: 566px;
+    height: 148px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    margin-left: 142px;
+  }
+
+  .get {
+    margin-left: 142px;
+  }
+
+  .divider {
+    width: 1156px;
+    margin-left: 142px;
+  }
+
+  .skills {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 130px 0;
+    gap: 152px;
+    position: absolute;
+    width: 1157.46px;
+    height: 596px;
+    left: 141px;
+    top: 720px;
+  }
+
+  .form {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 130px 0;
+    gap: 24px;
+    position: absolute;
+    width: 1163px;
+    height: 590px;
+    left: 135px;
+    top: 1400px;
+  }
+
+  .primery-text {
+    margin: 30px 0 0 12px;
+    width: 400px;
+    height: 256px;
+    text-align: left;
+  }
+
+  .contact-form {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 24px;
+    width: 684px;
+    height: 130px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }

```

Comment:
- [ ] Kindly fix your width on your desktop version, since in each section there's a scrollbar that can go to the right of the screen, in a responsive application you shouldn't be able to scroll to the right or left

![image](https://user-images.githubusercontent.com/93630700/208183172-9cb07bfc-dcec-4299-90e6-a430dcf229f9.png)